clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/ui/services/OutputProvider.kt

```diff
@@ -65,4 +65,8 @@ class OutputProvider(val project: Project) : Disposable {
             }
         }
     }
+
+    companion object {
```

Comment:
Do you need this object?
Do you use it for sync?

Reply:
fixed